person.rb

```diff
@@ -3,7 +3,7 @@
 require 'trimmer_decorator'
 
 class Person < Nameable
```

Comment:
- [ Required ] It would be better to set up an `add_rental` method to the `Person` class to set up the `A person` has `many rentals` association. This method can be used anytime `A teacher` or `Student` wants to rent a book, You can achieve this by adding the method below in the `Person` class.

```ruby
  def add_rental(book, date)
      Rental.new(date, book, self)
  end
```